pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -218,7 +223,15 @@ func configureFromProviderConfig(pushCtx *model.PushContext, proxy *model.Proxy,
 			}
 			return otelConfig(serviceCluster, hostname, clusterName)
 		}
-
+		// Providers without any tracing support
+	case *meshconfig.MeshConfig_ExtensionProvider_EnvoyExtAuthzHttp,
+		*meshconfig.MeshConfig_ExtensionProvider_EnvoyExtAuthzGrpc,
+		*meshconfig.MeshConfig_ExtensionProvider_EnvoyHttpAls,
+		*meshconfig.MeshConfig_ExtensionProvider_EnvoyTcpAls,
+		*meshconfig.MeshConfig_ExtensionProvider_EnvoyOtelAls,
+		*meshconfig.MeshConfig_ExtensionProvider_EnvoyFileAccessLog,
+		*meshconfig.MeshConfig_ExtensionProvider_Prometheus:
+		return nil, nil, fmt.Errorf("provider %T does not support tracing", provider)
```

Comment:
would be better add a metric for this? or reuse exists one.

Reply:
Not sure we have an existing one